deploy/cloud/scripts/init.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -e
+
+cloudDomain="cloud.io"
+tlsCrtPlaceholder="<tls-crt-placeholder>"
+tlsKeyPlaceholder="<tls-key-placeholder>"
+
+function read_env {
+  source $1
+}
+
+function mock_tls {
+  if grep -q $tlsCrtPlaceholder manifests/tls-secret.yaml; then
+    echo "mock tls secret"
+  else
+    echo "tls secret is already set"
+    return
+  fi
+
+  mkdir -p etc/tls
+  openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout etc/tls/tls.key -out etc/tls/tls.crt -subj "/CN=$1" -addext "subjectAltName=DNS:*.$1" > /dev/null
+  sed -i -e "s;$tlsCrtPlaceholder;$(base64 -w 0 etc/tls/tls.crt);" -e "s;$tlsKeyPlaceholder;$(base64 -w 0 etc/tls/tls.key);" manifests/tls-secret.yaml
+}
+
+function sealos_run_controller {
+  # run user controller
+  sealos run sealos.hub:5000/labring/sealos-cloud-user-controller:dev
+  # \ 1 > /dev/null
+
+  # run terminal controller
+  sealos run sealos.hub:5000/labring/sealos-cloud-terminal-controller:dev --env cloudDomain=$cloudDomain --env userNamespace="user-system" --env wildcardCertSecretName="wildcard-secret" --env wildcardCertSecretNamespace="sealos-system"
+  # \ 1 > /dev/null
+
+  # run app controller
+  sealos run sealos.hub:5000/labring/sealos-cloud-app-controller:dev
+  # \ 1 > /dev/null
+}
+
+function sealos_run_service {
+  # run auth service
+  sealos run sealos.hub:5000/labring/sealos-cloud-auth-service:dev --env cloudDomain=$cloudDomain --env certSecretName="wildcard-secret" --env callbackUrl="$cloudDomain/login/callback" --env ssoEndpoint="login.$cloudDomain" --env casdoorMysqlRootPassword="$(tr -cd 'a-z0-9' </dev/urandom | head -c16)"
+  # \ 1 > /dev/null
+}
+
+function sealos_run_frontend {
+  sealos run sealos.hub:5000/labring/sealos-cloud-desktop-frontend:dev --env cloudDomain=$cloudDomain --env certSecretName="wildcard-secret"
+
+  sealos run sealos.hub:5000/labring/sealos-cloud-applaunchpad-frontend:dev --env cloudDomain=$cloudDomain --env certSecretName="wildcard-secret"
+
+  sealos run sealos.hub:5000/labring/sealos-cloud-terminal-frontend:dev --env cloudDomain=$cloudDomain --env certSecretName="wildcard-secret"
+}
+
+
+function install {
+  # read env
+  read_env etc/sealos/cloud.env  # mock tls
+  mock_tls $cloudDomain
+  # kubectl apply namespace and secret
+  kubectl apply -f manifests
+  # sealos login
+  sealos login -u admin -p passw0rd sealos.hub:5000
```

Comment:
move password from shell to env? for temporary fix.

Reply:
I will fix this latter in next pr.